app/workers/import_bank_data_worker.rb

```diff
@@ -3,12 +3,17 @@ class ImportBankDataWorker
   include Sidekiq::Status::Worker
   sidekiq_options retry: false
 
-  def perform(applicant_id, token, token_expires_at)
+  attr_reader :legal_aid_application_id
+
+  def perform(legal_aid_application_id)
+    @legal_aid_application_id = legal_aid_application_id
     command = TrueLayer::BankDataImportService.call(
-      applicant: Applicant.find(applicant_id),
-      token: token,
-      token_expires_at: token_expires_at
+      legal_aid_application: legal_aid_application
     )
     store errors: command.errors.to_a.flatten.to_json unless command.success?
   end
+
+  def legal_aid_application
+    @legal_aid_application ||= LegalAidApplication.find(legal_aid_application_id)
```

Comment:
Unless `legal_aid_application_id` is called from outside this class, this and the `attr_reader` is a bit useless in my opinion. Is this just to avoid using a variable in the `perform` method?
Is the `@legal_aid_application ||=` memoization pattern useful here?